.github/sync-repo-settings.yaml

```
@@ -5,9 +5,8 @@ branchProtectionRules:
 - pattern: main
   isAdminEnforced: true
   requiredStatusCheckContexts:
-    - 'PHP 7.4 Unit Test'
-    - 'PHP 7.4 Unit Test protobuf,grpc'
     - 'PHP 8.0 Unit Test'
+    - 'PHP 8.0 Unit Test protobuf,grpc'
```

Also add "prefer lowest" tests
```suggestion
    - 'PHP 8.0 Unit Test protobuf,grpc'
    - 'PHP 8.0 Unit Test --prefer-lowest'
```